cpp/include/cudf/io_functions.h

```diff
@@ -63,11 +65,13 @@ gdf_error read_parquet(pq_read_arg *args);
  */
 gdf_error gdf_to_csr(gdf_column **gdfData, int num_cols, csr_gdf *csrReturn);
 
+namespace cudf {
 /*
  * @brief Reads JSON-structured data and returns an array of gdf_columns.
  *
  * @param[in,out] args Structure containing input and output arguments
  *
  * @return gdf_error GDF_SUCCESS if successful, otherwise an error code.
  */
-gdf_error read_json(json_read_arg *args);
+cudf::table *read_json(json_read_arg const *args);
```

Comment:
I think the return should be by value.

Reply:
I initially tried returning by value, but had some trouble declaring a cudf::table object on the stack in Cython. I'll give it another try.